package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "js-idm-wallet",
+  "version": "0.0.0",
+  "description": "The reference implementation of the IDM Wallet in JavaScript",
+  "main": "index.js",
+  "homepage": "https://github.com/ipfs-shipyard/js-idm-wallet#readme",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ipfs-shipyard/js-idm-wallet.git"
+  },
+  "keywords": [
+    "idm",
+    "wallet",
+    "js",
+    "javascript",
+    "sdk"
+  ],
+  "bugs": {
+    "url": "https://github.com/ipfs-shipyard/js-idm-wallet/issues"
+  },
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "typechecking": "tsc",
+    "test": "jest",
+    "build": "tsc && babel src --out-dir lib --extensions \".ts\"",
```

Comment:
Worth compiling to es too

Reply:
After some thought we decided to not use TypeScript after all.